src/utility/storage.ts

```diff
@@ -0,0 +1,24 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+class StorageManager {
```

Comment:
Why creating class?

Reply:
So that i can access storage methods from storage file
for now it has only two static methods but i will add new methods in it later